src/app/index.js

```diff
@@ -67,43 +67,43 @@ class App extends Component {
                         icon={<DashboardIcon />}
                         name="dashboard"
                         label="Dashboard"
-                        onClick={() => fetchSidebarState('/dashboard')} />
+                        onClick={() => toogleSidebar('/dashboard')} />
```

Comment:
we need to make two actions
- toggleSidebar (fix current is set to toogle but is toggle)
- navigate
then navigate call toggleSidebar
here use navigate and in the SectionHeading use toggleSidebar